src/main/java/primeNumbers/PrimeNumbersDefinition.java

```diff
@@ -0,0 +1,23 @@
+package primeNumbers;
+
+import java.util.*;
+
+public class PrimeNumbersDefinition {
+    private int numbersCounter = 0;
+    private List<Integer> primeNumbersList = new ArrayList<>();
+
+    public List<Integer> getPrimeNumbers(int to) {
+        for (int i = 2; i <= to; i++) {
+            for (int j = i; j > 0; j--) {
```

Comment:
in order to avoid not needed iterations - i suggest decreasing the range by 1 for both sides...

Reply:
Done in e10c88a644d8e317f6eebafd5b061a0e6df590d7.
The range in the 2nd loop is decreased by 1 from both sides.
'for' loops are now changed to stream iterators.